ansible/openshift-playbook.yml

```diff
@@ -11,6 +11,12 @@
     test_harness_run_as_root: true
     test_harness_node_selector:
       system-roles-ci: "true"
+    test_harness_dockerfile_from: "docker.io/library/fedora:32"
+    test_harness_build_config_files: []
```

Comment:
Is it ok to be empty? Or the variable will be overwritten by the inventory or command line separately?

Reply:
Right - and vars defined in the `vars` section here can be overridden by inventory or `-e` cmdline.